package.json

```diff
@@ -19,9 +19,12 @@
     "@angular/compiler": "~12.2.0",
     "@angular/core": "~12.2.0",
     "@angular/forms": "~12.2.0",
+    "@angular/localize": "~12.2.0",
```

Comment:
What's the strategy behind adding the package?

Reply:
Angular documentation says that it is used to prepare the project for usage in different locales (formatting dates, currencies, and time). I did not personally implement this so I will be leaving it in for the mean time.